config/seata-config-core/src/main/java/io/seata/config/Configuration.java

```diff
@@ -16,15 +16,19 @@
 package io.seata.config;
 
 import java.time.Duration;
+import java.util.Map;
 import java.util.Set;
 
+import io.seata.common.util.StringUtils;
+
 /**
  * The interface Configuration.
  *
  * @author slievrly
  */
 public interface Configuration {
 
+    Map<String, String> ENV_MAP = System.getenv();
```

Comment:
System.getProperties() 也支持一下吧？

Reply:
1. Env is immutable, so there is no setEnv method and it can use static variable global reference.

2. System.getProperty() is runtime mutable. It can replace a property with System.setProperty, so you can't have a static global reference.